test/unit/host_test.rb

```diff
@@ -1498,6 +1498,20 @@ def teardown
       assert_nothing_raised(ActiveRecord::InvalidForeignKey) {h.reload.destroy}
     end
 
+    test "token_expired? should be true if expiration date is in the past" do
```

Comment:
Can you write an expectation instead of writing the setting to the db? Same goes for the host creation - if you just want to test `.token_expired?` you can do with just 'FactoryGirl.build_stubbed(:host, :managed)` which doesn't save anything to db. 

The 2 first lines of code in this test and the following are the same, please move them to a setup block.